pulp_puppet_extensions_admin/pulp_puppet/extensions/admin/pulp_cli.py

```diff
@@ -7,7 +7,7 @@
 
 from pulp_puppet.common import constants
 from pulp_puppet.extensions.admin import structure
-from pulp_puppet.extensions.admin.consumer import bind, content
+from pulp_puppet.extensions.admin.consumer import bind
```

Comment:
Since the agent is going away, I don't expect us to support binding a repository to a consumer